doc/source/user_guide/make_container_image.rst

```diff
@@ -63,13 +63,13 @@ Run Docker container using the command line
 
 When you run the Docker container, you must specify the Ansys license file.
 
-Execute this command to run the Docker container in solver mode:
+To launch the container in solver mode, use:
```

Comment:
@hpohekar In this file we have one or two of the very few remaining uses of the phrase, "solver mode". Generally we prefer "solution mode" in line with the official Ansys documentation. Please could you update the last very few instances. Please be aware of a potential underlining length mismatch issue in one case when searching/replacing globally. Thanks.

Reply:
Done. Thank you.